src/literature/components/PublicationInfoList.jsx

```diff
@@ -12,6 +12,9 @@ class PublicationInfoList extends Component {
       <InlineList
         label="Published in"
         items={publicationInfo}
+        extractKey={info =>
+          info.get('journal_title') || info.get('pubinfo_freetext')
```

Comment:
why not ``artid``?

Reply:
because these are the main props of `publication_info`. If you see the logic in `PublicationInfo`, it only renders if one of them is actually present.